### PR TITLE
Xtext 2.23 release notes

### PR DESCRIPTION
**Pull request by kthoms**

### CODE REVIEW

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- Nico Prediger (itemis)
+- Sebastian Zarnekow (independent)
+- Karsten Thoms (Karakun)
+- Tamas Miklossy (itemis)
+- Arne Deutsch (itemis)
+- Jan Koehnlein (TypeFox)
+- Neeraj Bhusare (Zafin)
+- Titouan Vervack (Sigasi)
+- Hendrik Eeckhaut (Sigasi)
+- Frank Benoit ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:75]: ETAS

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Aaron R Miller ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
+- ivaniesta14 ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:77]: Ivan Nieves

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+* [Closed Pull Requests](https://github.com/search?utf8=%E2%9C%93&q=is%3Apr+milestone%3ARelease_2.23+is%3Aclosed+repo%3Aeclipse%2Fxtext+repo%3Aeclipse%2Fxtext-core+repo%3Aeclipse%2Fxtext-lib+repo%3Aeclipse%2Fxtext-extras+repo%3Aeclipse%2Fxtext-eclipse+repo%3Aeclipse%2Fxtext-idea+repo%3Aeclipse%2Fxtext-web+repo%3Aeclipse%2Fxtext-maven+repo%3Aeclipse%2Fxtext-xtend&type=Issues&ref=searchresults)
+
+* [Fixed Eclipse Bugzilla tickets](https://bugs.eclipse.org/bugs/buglist.cgi?bug_status=RESOLVED&bug_status=VERIFIED&bug_status=CLOSED&classification=Modeling&classification=Tools&columnlist=product%2Ccomponent%2Cassigned_to%2Cbug_status%2Cresolution%2Cshort_desc%2Cchangeddate%2Ckeywords&f0=OP&f1=OP&f3=CP&f4=CP&known_name=Xtext%202.23&list_id=16618269&product=TMF&product=Xtend&query_based_on=Xtext%202.23&query_format=advanced&status_whiteboard=v2.22&status_whiteboard_type=allwordssubstr)
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:88]: wrong whiteboard

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+The Xtext project is thankful for the dedication of each committer and contributor. This release has been made possible by the following persons (in order of the number of contributed commits to this release to [all repositories](https://github.com/eclipse/xtext#repositories) except [xtext-xtend](https://github.com/eclipse/xtext-xtend)):
+
+- Christian Dietrich (itemis)
+- Nico Prediger (itemis)
+- Sebastian Zarnekow (independent)
+- Karsten Thoms (Karakun)
+- Tamas Miklossy (itemis)
+- Arne Deutsch (itemis)
+- Jan Koehnlein (TypeFox)
+- Neeraj Bhusare (Zafin)
```
Comment (tivervac) [inline xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:72]: Not sure if this is conventional but we might want to do a small shoutout that Neeraj has become an official commiter

Comment (kthoms) [reply xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md:72]: We did not announce new committers in release notes in the past, but more in blog posts. And just formally, the nomination process did not conclude yet. Of course, his addition is highly appreciated.

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md (hunk 1 above)
Comment (kthoms) [reply xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md]: added

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md (hunk 2 above)
Comment (kthoms) [reply xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md]: changed

File: xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md (hunk 3 above)
Comment (kthoms) [reply xtext-website/_posts/releasenotes/2020-09-01-version-2-23-0.md]: updated to 2.23

**Review by cdietrich — APPROVED** [review]

**Pull request merged**